libcontainer/container_linux.go

```diff
@@ -1060,17 +1060,12 @@ func (c *linuxContainer) currentState() (*State, error) {
 		ExternalDescriptors: externalDescriptors,
 	}
 	if pid > 0 {
-		for _, ns := range c.config.Namespaces {
-			state.NamespacePaths[ns.Type] = ns.GetPath(pid)
-		}
```

Comment:
You are right, people can set it's own namespace path in `c.config.Namespaces` instead of using `/proc/$pid/ns`, thanks.